Draggable.js

```diff
@@ -150,11 +150,14 @@ export default class Draggable extends Component {
 		};
 	}
 	_getTextOrImage = () => {
-		const { renderSize, renderShape, renderText, imageSource } = this.props;
+		const { renderSize, renderShape, renderText, imageSource, children } = this.props;
 		if(renderShape == 'image') {
```

Comment:
This can be better. It seems we don't need nested if condition.

for ex:
if(renderShape == 'image') { 
...
} else if(renderText) {
...
} else if(renderChildren) {
...
}